src/components/main/MainCard.tsx

```diff
@@ -58,9 +60,14 @@ const MainCard: FC<MainCardProps> = ({ currentWeather, location }) => {
     setIsFavorite(getIsFavorite(location.id));
   }, [location.id]);
 
+  const mainCardClasses = useMemo(
+    () => getClasses('main-card__item', 'main-card__item_center'),
+    []
+  );
+
   return (
     <div className={styles['main-card']}>
-      <div className={`${styles['main-card__item']} ${styles['main-card__item_center']}`}>
+      <div className={mainCardClasses}>
```

Comment:
just interesting - doesn't classnames work without binding like this?
`<div className={classnames(styles['main-card__item'], styles['main-card__item_center'])}>`

Reply:
work;)
fix it?

Reply:
yes, no need in overengineering